proto/toit/api/app.proto

```diff
@@ -36,12 +37,39 @@ message CreateAppResponse {
 
 message GetAppRequest {
   bytes app_id = 1;
+  // if revision is zero will give the latest.
+  uint64 revision = 2;
 }
 
 message GetAppResponse {
   model.App app = 1;
 }
 
+message GetAppFilesRequest {
+  bytes app_id = 1;
+  // if revision is zero will give the latest.
+  uint64 revision = 2;
+  // If path ends with '/', the request is for a directory.
+  string path = 3;
+}
+
+message GetAppFilesResponse {
+  message Entry {
+    string name = 1;
+    // content_type only set if not a directory
+    string content_type = 2;
```

Comment:
ENUM? Just to restrict it a bit, and then have an UNKNOWN_FILE or something...

Reply:
since we want to save all kind of files the enum will be very long.